app/domain/urls.ts

```diff
@@ -1,6 +1,6 @@
 import config from '../config';
 
-export type Locale = 'no' | 'nl' | 'en' | 'sk' | 'it' | 'tr';
+export type Locale = 'no' | 'nl' | 'en' | 'es-mx' | 'sk' | 'it' | 'tr';
```

Comment:
Don't know if it matters, but we could consider keeping the casing consistent with i18n locales: https://github.com/ladjs/i18n-locales